src/webroot/css/cake-core.min.css

```diff
@@ -0,0 +1 @@
+.cake-core-hidden{display:none}
```

Comment:
FormHelper::postLink() is just one method which has to be changed for CSP support, I don't know what style classes have to be added in future. Therefore also a min file. :wink: